core/libdeno.rs

```diff
@@ -85,6 +55,10 @@ pub struct DenoIsolate {
   dyn_import_cb_: deno_dyn_import_cb,
   dyn_import_map_: HashMap<deno_dyn_import_id, v8::Global<v8::PromiseResolver>>,
   pending_promise_map_: HashMap<i32, v8::Global<v8::Value>>,
+  // Used in deno_mod_instantiate
+  resolve_context_: *mut c_void,
+  // Used in deno_mod_evaluate
+  core_isolate_: *mut c_void,
```

Comment:
Is it possible to use non-pointer types?

Reply:
These are references so it won't be easily done without adding lifetimes, anyway, both of these fields will be removed shortly so let's just roll with it?